src/components/Button/Button.js

```diff
@@ -2,7 +2,7 @@ import React, { useRef, forwardRef } from "react";
 import PropTypes from "prop-types";
 import cn from "classnames";
 import Spinner from "~/components/Spinner";
-import Link from "~/components/Link";
+import Link from "../Link";
```

Comment:
**suggestion** WDYT about keeping the import statements consistent? 

Here we import the `Spinner` with an alias and the `Link` component relatively. They are in the same location so we can import them the same way. I'd favor the one with alias as in the future if we relocate this component elsewhere we won't have to update the import statements. 

But I don't have strong objections against the relative import. In that case I'd suggest to change the `Spinner` import to `relative` import as well.

Reply:
ok, about the ~, we can figure out because now it can be handled by the alias of tsconfig

Reply:
Personally I'm more fond of the `~` import. Especially now that with `tsconfig` the IDE can understand it :)